src/opm/input/eclipse/Schedule/Well/Connection.cpp

```diff
@@ -509,41 +597,35 @@ void Connection::setInjMult(const InjMult& inj_mult) {
     m_injmult = inj_mult;
 }
 
-
 void Connection::setFilterCake(const FilterCake& filter_cake) {
```

Comment:
any reason you stopped moving open braces to a new line here?

Reply:
> any reason you stopped moving open braces to a new line here?

No, not really.  I've pushed an update to make that change (function's opening brace on the next line) throughout the file.  Now I "just" need to figure out why the [parallel regression test](https://ci.opm-project.org/job/opm-common-PR-builder/lastCompletedBuild/testReport/(root)/mpi/compareParallelSim_flow_WINJDAM_MSW/) fails.